### PR TITLE
How to quickly set up the header

### PR DESCRIPTION
**Pull request by gersongcg**
# Motivation

Steps and configuration required to set up the theme header.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,4 +35,42 @@ The following templates will override the out-of-the-box theme templates.
...
 
-**Compatible with StoreConnect v16.0.0 and on**
\ No newline at end of file
+### Setting up the Header
+
+**Top header**
+
+This is the white section at the top of your header and it features a small slideshow on the left for short content, with two customizable links on the right.
+
+If this component is not needed, you can easily deactivate it by setting the `active.top.header` theme variable to `inactive`, `off`, or any other string.
```
Comment (toddweetman) [inline README.md:44]: Check this is true:

```suggestion
If this component is not needed, you can easily deactivate it by setting the `active.top.header` theme variable to `inactive`, `off`, or any string other than `active`.
```

File: README.md [hunk 2]
```diff
@@ -35,4 +35,42 @@ The following templates will override the out-of-the-box theme templates.
...
+
+**Help Centre**
+
+By default, the theme sets the help centre label and placeholder link. You can relate this link to anything you want, an article, customer support, product category, you name it.
+
+To get familiar with this feature follow these steps:
+
+1. Create a page
+2. Copy the path
+3. Go to the theme variables and find the `top.slides.cb.identifier` variable
```
Comment (toddweetman) [inline README.md:69]: this is the same variable as used by the Top head slider. Are they not separate features?

**Review by toddweetman — CHANGES_REQUESTED** [review]
May be useful to include a small screenshot of the feature you are talking about to give context.

File: README.md (hunk 1 above)
Comment (gersongcg) [reply README.md:44]: @toddweetman Yes, I have verified it and it works.

File: README.md (hunk 2 above)
Comment (gersongcg) [reply README.md:69]: @toddweetman nope, my bad, I correct this bit.

**Review by toddweetman — APPROVED** [review]

**Pull request merged**